app/models/account/provider_methods.rb

```diff
@@ -202,6 +202,10 @@ def authentication_provider_kinds
     indexed.values.flatten.sort_by{ |a| a.to_param.to_s }.reverse
   end
 
+  def provider_multiservice?(scope = {})
```

Comment:
How about we call this method `multiple_accessible_services?` as that seems more descriptive to me.